src/Stratis.Bitcoin.Features.Consensus/ProvenBlockHeaders/ProvenBlockHeaderStore.cs

```diff
@@ -231,9 +264,9 @@ public async Task SaveAsync()
 
                 lock (this.lockObject)
                 {
-                    pendingBatch = new SortedDictionary<int, ProvenBlockHeader>(this.PendingBatch);
+                    pendingBatch = new SortedDictionary<int, ProvenBlockHeader>(this.pendingBatch);
```

Comment:
After the rename now there are 2 variables with the name `pendingBatch`
Please rename one of them to avoid confusion

Reply:
Yeah fixed